stellar_account_prometheus_exporter/exporter.py

```diff
@@ -62,6 +65,9 @@ def do_GET(self):
         m_num_sponsoring = Gauge("stellar_account_num_sponsoring", "Stellar core account number of sponsoring entries",
                                  account_label_names, registry=self.registry)
 
+        m_account_exists = Gauge("stellar_account_success", "Displays whether or not stellar core account is present",
```

Comment:
This is not entirely correct. There are error conditions, like horizon being down, that will cause errors and this metric will be 0 even if the account exists. I think it would be best to call it something generic so that it clear what it is.
```suggestion
        m_account_exists = Gauge("stellar_account_scrape_success", "Indicates whether data was gathered successfully",
```